source/base/Debug.ooc

```diff
@@ -10,11 +10,9 @@ import io/FileWriter
 
 DebugLevel: enum {
 	Everything
+	Message
 	Debug
-	Notification
 	Warning
```

Comment:
To many levels is just confusing in my opinion, hence, I put my vote on not having "Error". It should be possible to place such stuff in either warning or critical.